definition_of_ready.md

```diff
@@ -0,0 +1,34 @@
+# Definition of ready (DoR)
+- Each user story should fulfill the INVEST criteria (see e.g. [Wikipedia](https://en.wikipedia.org/wiki/INVEST_(mnemonic)), [scrum-in-der-praxis.de](https://scrum-in-der-praxis.de/glossary/invest))
+  - See https://www.knowledgehut.com/tutorials/scrum-tutorial/definition-of-ready
+    - Independent (of all others)
+    - Negotiable (not a contract for a specific feature)
+    - Valuable (a vertical slice)
+    - Estimable (to a decent approximation)
+    - Small (enough to fit in a single iteration)
+    - Testable (even if the test doesn't exist yet)
+
+## Further requirements
+- The user story is Well-defined:
```

Comment:
What about non-user-stories? Issues? Tutorials?

Reply:
Imo everything can be written as a user-story. (whether that makes sense is another point of discussion though)